messages/sec.options

```diff
@@ -1,3 +1,3 @@
 orb.mcu.sec.SERequest.data max_size: 512
 orb.mcu.sec.SEResponse.data max_size: 512
-orb.mcu.sec.Tamper.unencrypted_cbor max_size: 512
+orb.mcu.sec.Tamper.unencrypted_cbor max_size: 58
```

Comment:
@sri9311 I maximized the size of the message. the goal being that the message fits into a CAN-FD frame (64-bytes)